lib/xgu/xgu.h

```diff
@@ -0,0 +1,691 @@
+#ifndef __XGU_H
+#define __XGU_H
+
+#include <assert.h>
+#include <pbkit/pbkit.h>
+// Hack until we fix pbkit/outer.h (or stop including it)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmacro-redefined"
+#include "nv2a_regs.h"
+#pragma GCC diagnostic pop
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <xboxrt/libc_extensions/strings.h>
```

Comment:
This will have to be changed once pdclib is updated

Reply:
Extra include removed, `<string.h>` seems to be enough.